server/app/lib/fusor/manifest/manifest_importer.rb

```diff
@@ -0,0 +1,64 @@
+#
+# Copyright 2015 Red Hat, Inc.
+#
+# This software is licensed to you under the GNU General Public
+# License as published by the Free Software Foundation; either version
+# 2 of the License (GPLv2) or (at your option) any later version.
+# There is NO WARRANTY for this software, express or implied,
+# including the implied warranties of MERCHANTABILITY,
+# NON-INFRINGEMENT, or FITNESS FOR A PARTICULAR PURPOSE. You should
+# have received a copy of GPLv2 along with this software; if not, see
+# http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+
+require 'zip'
+require 'fileutils'
+require 'json'
+
+module Fusor
+  module Manifest
+    class ManifestImporter
+
+      def parse_entitlement_json(ent_json)
+        # stuff
+        entitlement = JSON.parse(ent_json)
+        Rails.logger.debug "------------------------------------"
+        Rails.logger.debug "Subscription Name: #{entitlement['pool']['productName']}"
+        Rails.logger.debug "Subscription Name2: #{entitlement['pool']['branding'].first['name']}"
+        Rails.logger.debug "Contract Number: #{entitlement['pool']['contractNumber']}"
+        Rails.logger.debug "System Type: N/A"
+        Rails.logger.debug "Start Date: #{entitlement['startDate']}"
+        Rails.logger.debug "End Date: #{entitlement['endDate']}"
+        Rails.logger.debug "Attached: #{entitlement['quantity']}"
+        available = entitlement['pool']['quantity'] - entitlement['quantity']
```

Comment:
I'd probably move this line up to after where you define `entitlement`. It's easier to spot that way.